arch/arm/src/tiva/common/lmxx_tm4c_start.c

```diff
@@ -53,9 +53,7 @@
  * ARM EABI requires 64 bit stack alignment.
  */
 
-#define IDLE_STACKSIZE (CONFIG_IDLETHREAD_STACKSIZE & ~7)
```

Comment:
@xiaoxiang781216 this is what I am talking about. Won't the heap base be wrong if the stack size config variable is not on the right boundary.

Reply:
Oh, you are talking about the idle thread stack. The heap manager will align the address for us:
https://github.com/apache/incubator-nuttx/blob/master/mm/mm_heap/mm_initialize.c#L101-L102
So, we need pass the real unaligned heap base(stack end), and the heap manager will move the address to the next aligned boundary.